lib/ansible/constants.py

```diff
@@ -236,6 +236,7 @@ def load_config_file():
 DEFAULT_INTERNAL_POLL_INTERVAL = get_config(p, DEFAULTS, 'internal_poll_interval', None, 0.001, value_type='float')
 ERROR_ON_MISSING_HANDLER  = get_config(p, DEFAULTS, 'error_on_missing_handler', 'ANSIBLE_ERROR_ON_MISSING_HANDLER', True, value_type='boolean')
 SHOW_CUSTOM_STATS = get_config(p, DEFAULTS, 'show_custom_stats', 'ANSIBLE_SHOW_CUSTOM_STATS', False, value_type='boolean')
+DEFAULT_ROLE_SKIP_DEPENDENCIES    = get_config(p, DEFAULTS, 'role_skip_dependencies', 'ROLE_SKIP_DEPENDENCIES', False, value_type='boolean')
```

Comment:
sounds good to me. Which would need to update lib/ansible/playbook/role/__init__.py too